src/components/content/content-test.stories.tsx

```diff
@@ -53,15 +53,3 @@ DefaultStory.args = {
   titleWidth: "",
   bodyFullWidth: false,
 };
-
-export const ContentComponentTest = ({
```

Comment:
**suggestion**: You can add this back, there's not much reason for it to be deleted. Keeping the test story means people can still use it to test locally.